examples/mapreader_plant_scivision.ipynb

```diff
@@ -0,0 +1,153 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9bc9f3b5",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "!pip install scivision"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5e355c21",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# solve issue with autocomplete\n",
+    "%config Completer.use_jedi = False\n",
+    "\n",
+    "%load_ext autoreload\n",
+    "%autoreload 2\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e0cc71b2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from scivision.io import load_pretrained_model, load_dataset\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8244a270",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cat = load_dataset('https://github.com/alan-turing-institute/mapreader-plant-scivision')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "559e8c83",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "list(cat)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "190769c4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ds = cat.plant_single().to_dask()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "51df9860",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.imshow(ds[0])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f809538e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scivision_yml = 'https://raw.githubusercontent.com/alan-turing-institute/mapreader-plant-scivision/main/.scivision-config.yaml'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8a545262",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = load_pretrained_model(scivision_yml, allow_install=True)"
```

Comment:
When I try to run this it still returns the following error, is this still something we need to find a solution for?:
`CalledProcessError                        Traceback (most recent call last)
[<ipython-input-14-c8b1561bc5b4>](https://localhost:8080/#) in <module>()
----> 1 model = load_pretrained_model(scivision_yml, allow_install=True)

4 frames
[/usr/lib/python3.7/subprocess.py](https://localhost:8080/#) in check_call(*popenargs, **kwargs)
    361         if cmd is None:
    362             cmd = popenargs[0]
--> 363         raise CalledProcessError(retcode, cmd)
    364     return 0
    365 

CalledProcessError: Command '['/usr/bin/python3', '-m', 'pip', 'install', 'git+https://github.com/alan-turing-institute/mapreader-plant-scivision@main#egg=mapreader-plant-scivision']' returned non-zero exit status 1.`